src/network-manager/azext_network_manager/_params.py

```diff
@@ -536,3 +541,155 @@ def load_arguments(self, _):
         c.argument('flag', type=str, help='Default rule flag.')
         c.argument('kind', type=str, help='Required. Whether the rule is custom or default.Constant filled by server.', arg_type=get_enum_type(['Custom', 'Default']))
         c.ignore('user_rule')
+
+    with self.argument_context('network manager connection create') as c:
```

Comment:
Network manager connections can either be created at the subscription or management group level. I think the commands should either be 
'network manager connection subscription create' and 'network manager connection management-group create'
or if its possible, allow the user to pass --management-group-id or --subscriptionId, and whichever flag they pass will determine the api that is called